cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+    cy.get(':nth-child(1) > .form-control')
+      .type('luckezcy@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('testPassword1!');
+    cy.get('.btn')
+      .click('bottom');
+    cy.get(':nth-child(4) > .nav-link')
```

Comment:
Again, using ':nth-child' as a selector is not recommended. This test might fail if another element is added to the navigation bar. Try to use a more specific and descriptive selector.